common/src/main/java/com/wynntils/core/config/ConfigHolder.java

```diff
@@ -62,14 +70,14 @@ public String getDisplayName() {
         if (!getMetadata().key().isEmpty()) {
             return I18n.get(getMetadata().key() + ".name");
         }
-        return parent.getTranslation(field.getName() + ".name");
```

Comment:
Are we sure parent is Translatable? If so, maybe we should even declare it as such in the constructor.

Otherwise, nowadays there is a nice pattern matching instanceof operation:
`if (!parent instanceof Translatable translatabe) return;` and then you can use `translatable` as a correctly casted alias for parent.